data/src/main/java/com/pocs/data/paging/AdminPostByUserPagingSource.kt

```diff
@@ -16,20 +16,19 @@ class AdminPostByUserPagingSource @Inject constructor(
 
     companion object {
         private const val START_PAGE = 1
+        private const val PAGING_SIZE = 30
```

Comment:
아래처럼 public으로 바꾸고 repository들에서 하드코딩한 부분을 수정해주세요. 그리고 PAGING_SIZE보다 PAGE_SIZE가 좋아보이네요.
```suggestion
        const val PAGE_SIZE = 30
```